### PR TITLE
Fix mismatching columns in CSV export

### PR DESCRIPTION
**Pull request by Gaston69**
### Short description
Bugfix csv-export of weekly statistics 

### Proposed changes
- Ensure export labels don't contain comma and corrupt CSV 

### Resolved issues

Fixes: #1701

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Comment by Gaston69** [comment]
Peter solved it! :-)

**Comment by timoludwig** [comment]
> Peter solved it! :-)

If you want to give him credit, you can also do `git commit --amend` and add

```
Co-authored-by: Peter Nerlich <nerlich@integreat-app.de>
```
to the commit message :wink:

(after that, you have to do `git push --force-with-lease` to overwrite the existing commit)

**Review by PeterNerlich — APPROVED** [review]
Since it seems simple enough, I'll add the first approve myself. We looked into it with four eyes, after all.

**Review by timoludwig — COMMENTED** [review]
Thanks, this fixes the issue! :+1:

Could you add a changelog entry for this?

**Review by timoludwig — CHANGES_REQUESTED** [review]
Thanks 👍
I know what is meant by the changelog entry because I know the context of the issue, but I think others might need a bit more info what this change is about.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 UNRELEASED
 ----------
 
+* [ [#1701](https://github.com/digitalfabrik/integreat-cms/issues/1701) ] Fix mismatching columns on CSV export of weekly statistics report
```
Comment (PeterNerlich) [inline CHANGELOG.md:4]:
```suggestion
* [ [#1701](https://github.com/digitalfabrik/integreat-cms/issues/1701) ] Fix malformed CSV export on weekly statistics report
```

**Review by PeterNerlich — APPROVED** [review]

**Review by timoludwig — APPROVED** [review]

**Pull request merged**